install.sh

```diff
@@ -562,7 +562,7 @@ EOF
     cat <<-EOF >>"${repodir}/rancher-rke2.repo"
 [rancher-rke2-${rke2_majmin}-${rke2_rpm_channel}]
 name=Rancher RKE2 ${rke2_majmin} (${1})
-baseurl=https://${rpm_site}/rke2/${rke2_rpm_channel}/${rke2_majmin}/${rpm_site_infix}/x86_64
+baseurl=https://${rpm_site}/rke2/${rke2_rpm_channel}/${rke2_majmin}/${rpm_site_infix}/${ARCH}
```

Comment:
Actually I think you want to use `uname -m` as the $ARCH is a docker/golang arch, not the machine type as used by RPM. Ref: 
https://github.com/rancher/rke2/blob/bf1bcfdcbe0d37b8a6f64c25b47b7e096e6efedb/install.sh#L164-L167
```suggestion
baseurl=https://${rpm_site}/rke2/${rke2_rpm_channel}/${rke2_majmin}/${rpm_site_infix}/$(uname -m)
```

Reply:
Yes `uname -m` is the way to go